dali/operators/segmentation/random_mask_pixel.cc

```diff
@@ -78,11 +78,11 @@ class RandomMaskPixelCPU : public Operator<CPUBackend> {
 
 RandomMaskPixelCPU::RandomMaskPixelCPU(const OpSpec &spec)
     : Operator<CPUBackend>(spec),
-      seed_(spec.GetArgument<int64_t>("seed")),
+      rngs_(spec.GetArgument<int64_t>("seed"), spec.GetArgument<int64_t>("batch_size")),
```

Comment:
It won't play nicely with our recent movement through variable batch size.
How about the old approach, but:
```
auto &rng = rng_[sample_idx%thread_id];
```
So you still have only `thread_pool.size()` rngs, and have deterministic way of assigning them to the threads.

Reply:
It will place nicely since I am creating enough seeds to fit the maximum batch size.

Reply:
So be it.